README.md

```diff
@@ -2,42 +2,49 @@
 # Permit.io PDP
 The PDP (Policy decision point) syncs with the authorization service and maintains up-to-date policy cache for open policy agent.
 
-## Running locally (during development)
-```
-uvicorn horizon.main:app --reload --port=7000
-```
+## Running a PDP
+PDPs are connected to your [Permit.io account](https://docs.permit.io/quickstart) using an API Key.
+Check out the [Permit.io documentation](https://docs.permit.io/manage-your-account/projects-and-env#fetching-and-rotating-the-api-key) to learn how to get an Environment API Key.
 
-you can pass environment variables to control the behavior of the sidecar:
-e.g, running a local sidecar against production backend:
-```
-AUTHZ_SERVICE_URL=https://api.permit.io CLIENT_TOKEN=<CLIENT_TOKEN> uvicorn horizon.main:app --reload --port=7000
+You can run a PDP in a docker container by running the following command:
+```bash
+docker run -it -p 7766:7000 -e PDP_API_KEY=<YOUR_API_KEY> -e PDP_DEBUG=True permitio/pdp-v2:latest
 ```
 
-## Installing and running in production
+### Deploying PDP to Production
+You can deploy the PDP to production in multiple designs. See the [Permit.io documentation](https://docs.permit.io/concepts/pdp/overview) for more information.
 
-Pull the image from docker hub
-```
-docker pull permitio/pdp-v2
+## Contributing
+
+### Setting up the development environment
+1. Clone the repository
+2. Install the dependencies
+```bash
+pip install -r requirements-dev.txt
 ```
 
-Run the image: don't forget to pass your authorization service API KEY:
+### Running locally (during development)
 ```
-docker run -it -e "CLIENT_TOKEN=<YOUR API KEY>" -p 7000:7000 permitio/pdp-v2
+uvicorn horizon.main:app --reload --port=7000
 ```
 
-By default the image exposes port 7000 but you can change it.
+You can pass environment variables to control the behavior of the PDP image.
+For example, running a local PDP against the Permit API:
+```
+AUTHZ_SERVICE_URL=https://api.permit.io PDP_API_KEY=<YOUR_API_KEY> uvicorn horizon.main:app --reload --port=7000
```

Comment:
I don't think there is really an AUTHZ_SERVICE_URL - PDP_CONTROL_PLANE is the right one